### PR TITLE
Fixes some tool interactions

### LINKED ISSUE #35719: Gas Heaters/Freezers cannot be rotated via wrench
Really quite annoying.

I took a gander at it on the off chance that I could fix it, but no, it's got a snowflake 

`if(default_change_direction_wrench(user, I))
		return`

Basically, pop open with screw driver, attempt to use wrench, wrench doesn't rotate the unit. Crowbar deconstructs just fine, however.

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 
:cl: Dax Dupont
fix: You can rotate freezers and cryo again. 
/:cl:

[why]: @DaedalusGame 
Fixes #35719

### CODE REVIEW

**Review by Cyberboss — APPROVED** [review]

**Comment by Poojawa** [comment]
based Dax. <3

File: code/modules/atmospherics/machinery/atmosmachinery.dm [hunk 1]
```diff
@@ -188,7 +188,7 @@ Pipelines + Other Objects -> Pipe network
 
 /obj/machinery/atmospherics/wrench_act(mob/living/user, obj/item/I)
 	if(!can_unwrench(user))
-		return TRUE
+		return ..()
```
Comment (ShizCalev) [inline code/modules/atmospherics/machinery/atmosmachinery.dm]: Shouldn't this be if they -can- wrench it?

**Comment by DaxDupont** [comment]
No, can_unwrench is specifically for pipes and other pipe related things.
Since it's some snowflake thing.

File: code/modules/atmospherics/machinery/atmosmachinery.dm (hunk 1 above)
Comment (DaxDupont) [reply code/modules/atmospherics/machinery/atmosmachinery.dm]: Can unwrench is used solely by pipes, connectors and what not but not machines. It's a snowflake overwrite, but not doing that return there will block wrenches from doing anything ie:can't hit things with a wrench or other actions

**Pull request merged**